code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1210,13 +1210,37 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	switch(H.nutrition)
 		if(NUTRITION_LEVEL_FULL to INFINITY)
 			H.throw_alert("nutrition", /obj/screen/alert/fat)
+			var/obj/item/organ/O = H.getorganslot(ORGAN_SLOT_HEART)
+			O.applyOrganDamage(0.25)
 		if(NUTRITION_LEVEL_HUNGRY to NUTRITION_LEVEL_FULL)
 			H.clear_alert("nutrition")
+			for(var/slot in list(ORGAN_SLOT_BRAIN, ORGAN_SLOT_HEART, ORGAN_SLOT_LUNGS, ORGAN_SLOT_APPENDIX, \
+	ORGAN_SLOT_EYES, ORGAN_SLOT_EARS, ORGAN_SLOT_TONGUE, ORGAN_SLOT_LIVER))
+				var/obj/item/organ/O = H.getorganslot(slot)
+				O.applyOrganDamage(-0.25)
 		if(NUTRITION_LEVEL_STARVING to NUTRITION_LEVEL_HUNGRY)
 			H.throw_alert("nutrition", /obj/screen/alert/hungry)
 		if(0 to NUTRITION_LEVEL_STARVING)
 			H.throw_alert("nutrition", /obj/screen/alert/starving)
-
+			for(var/slot in list(ORGAN_SLOT_BRAIN, ORGAN_SLOT_HEART, ORGAN_SLOT_LUNGS, ORGAN_SLOT_APPENDIX, \
+	ORGAN_SLOT_EYES, ORGAN_SLOT_EARS, ORGAN_SLOT_TONGUE, ORGAN_SLOT_LIVER))
```

Comment:
I would recommend leaving it to a small number of organs, like heart, liver and stomach, since a wide list will make the damage fairly inconsequential. I don't necessarily agree that stomach damage should be avoided, since massive starvation caused by a damaged stomach should actually be a genuine medical emergency.